services/core-web/src/styles/components/NoticeOfWork.scss

```diff
@@ -144,7 +144,7 @@
     position: relative;
     padding-right: 20px;
     left: 250px;
-    width: calc(100vw - 250px);
+    width: calc(100vw - 270px);
```

Comment:
What was this change addressing?

Reply:
If we keep it at -250px, there is an unnecessary horizontal scrollbar that is created on the page. We need to account not only for left's 250px, but also the padding-right's 20px to remove this unnecessary scrollbar.